dom/events/delegate/enter-leave.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-var domEvents = require("../events");
+var domEvents = require("../events"),
```

Comment:
there's maybe one line that still resembles their code but I'll add the copyright just to be safe